hls4ml/model/optimizer/passes/multi_dense.py

```diff
@@ -47,6 +47,8 @@ def transform(self, model, node):
 
         class_name = 'PointwiseConv' + str(dim) + 'D'
         pw_node = model.make_node(class_name, node.name, pointwise_attrs, node.inputs.copy())
+        for key in node.weights:
+            pw_node.weights[key] = node.weights[key]
```

Comment:
Just to be futureproof, we should make sure the weights have [proper shape](https://github.com/fastmachinelearning/hls4ml/pull/366/files#diff-596f42295045305371ca3a443fce3a9c78302f2462ded2f2b16e2c3791944e95R661). Bias is fine.

Reply:
done in bdd1921